README.md

```
@@ -1,2 +1,98 @@
+1. Navigate to your FreshRSS extensions folder:
+  ```bash
+  cd /path/to/freshrss/extensions/
+  ```
+
+2. Clone this repository:
+  ```bash
+  git clone https://github.com/tryallthethings/freshvibes.git
+  ```
+
+3. In FreshRSS:
+  - Go to **Configuration** → **Extensions**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)